.gitignore

```
@@ -151,3 +151,5 @@ osx_volname
 dist/
 
 /guix-build-*
+
+ci/scratch/
```

nit: Would be more specific:
```suggestion
/ci/scratch/
```

---

done!